test/samples/with-preact/index.js

```diff
@@ -0,0 +1,16 @@
+/** @jsx h */
+
+import { props, withProps, withComponent } from '../../../src';
+import withRenderer from '@skatejs/renderer-preact';
+import { h } from 'preact';
+
+const Component = withComponent(withProps(withRenderer()));
```

Comment:
You don't need `withProps()` here. The `withComponent()` mixin already brings that in.